src/operator/quantization/dnnl/dnnl_quantized_transpose.cc

```diff
@@ -0,0 +1,102 @@
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file dnnl_quantized_transpose.cc
+ * \author: Rafal Litka, rafal.litka@intel.com
+ */
+#if MXNET_USE_ONEDNN == 1
+#include "../../numpy/np_matrix_op-inl.h"
+#include "../../tensor/matrix_op-inl.h"
+#include "../../nn/dnnl/dnnl_transpose-inl.h"
```

Comment:
```suggestion
#include "operator/numpy/np_matrix_op-inl.h"
#include "operator/tensor/matrix_op-inl.h"
#include "operator/nn/dnnl/dnnl_transpose-inl.h"
```

some time ago PR was merged to replace relative paths in dnnl files